webapp/src/app/[locale]/layout.tsx

```diff
@@ -37,10 +37,6 @@ async function getLanguages(locale: string) {
   }
 }
 
-export async function generateStaticParams() {
```

Comment:
@tetogomez @AngeloCG97 please check this.

⨯ Error: Page "/[locale]/page" cannot use both "use client" and export function "generateStaticParams()".